src/app/pages/system/advanced/init-shutdown/init-shutdown-card/init-shutdown-card.component.ts

```diff
@@ -84,18 +84,19 @@ export class InitShutdownCardComponent implements OnInit {
   ) {}
 
   ngOnInit(): void {
-    const scripts$ = this.ws.call('initshutdownscript.query').pipe(
-      untilDestroyed(this),
-    );
+    const scripts$ = this.ws.call('initshutdownscript.query');
     this.dataProvider = new AsyncDataProvider<InitShutdownScript>(scripts$);
+    this.loadScripts();
   }
 
   onAdd(): void {
     this.openForm();
   }
 
   loadScripts(): void {
-    this.dataProvider.refresh();
+    this.dataProvider.load<InitShutdownScript[]>(() => pipe(
```

Comment:
This looks interesting, but I think a more native approach would be to just return an observable and subscribe to it.

Reply:
this is addressed as well